hospital_simulator/.env.production

```diff
@@ -1,3 +1,2 @@
-NEXT_PUBLIC_FHIR_BASE_URL=http://localhost:9090/fhir
-NEXT_PUBLIC_FHIR_BASE_URL_DOCKER=http://fhirstore:8080/fhir
-NEXT_PUBLIC_ORCA_BASE_URL=/orca
\ No newline at end of file
+FHIR_BASE_URL=http://fhirstore:8080/fhir
```

Comment:
This is going to be depending on the deployment environment.

Reply:
Yeah, this will just be the default for the `Dockerfile` / dev deployment. When Ops overwrites the env var it should overwrite this value.